### ISSUE TITLE
When clicking on my own icon on the homescreen, I expect that I can edit my page as well

### ISSUE THREAD

**Issue by ACT-06** [selfdevs/accountability-dashboard#84]
Click this will result in me 'visiting' my own page whilst I am the owner of it
<img width="149" alt="Screenshot 2022-02-11 at 12 50 49" src="https://user-images.githubusercontent.com/6077854/153586673-ade597a4-c7a9-410d-b98d-972a180a2e1b.png">

**Comment by Juliotati** [comment]
Yeah thought of the same thing, but looks like the current implementation is set just for the availability of `public view` and if you want to edit your page you need to explicitly click edit my dashboard.

@cchampou maybe we can have the current user always be able to edit they dashboard and have a button to check the `public view` within their dashboard

**Comment by open-longlabs** [comment]
agreed. just need some condition flow here I think!

**Comment by cchampou** [comment]
Agree, this would make more sense. Anyone wanting to develop this just assign yourself to this ticket first please.

**Comment by probablyArth** [comment]
Ima get to it after #83  haha
